examples/kotlin/src/main/kotlin/com/example/booktest/postgresql/Models.kt

```diff
@@ -0,0 +1,32 @@
+// Code generated by sqlc. DO NOT EDIT.
+
+package com.example.booktest.postgresql
+
+import java.time.OffsetDateTime
```

Comment:
Wow I didn't actually know about this new DateTime class. I think we tend to use `LocalDate` and `Instant` but that's probably because we predate Java 8 and we always store timestamps with UTC. This does look like a good fit as a general timezoned datetime instance.

Reply:
I didn't either lol. The postgres jdbc driver explicitly doesn't support `Instant`